doctrine.rst

```diff
@@ -861,6 +861,41 @@ control behavior:
 
     The ``message`` option was introduced in Symfony 7.1.
 
+Mapped Route Parameters
+~~~~~~~~~~~~~~~~~~~~~~~
+
+When many route parameters are used to find more than one entity,
+it is mandatory to use ``#[MapEntity]`` attributes and this can become cumbersome::
+
+    #[Route('/document/{slug}/{id}-{name}/')]
```

Comment:
What do you think about removing the trailing slash from new examples for consistency with current documentation ?